src/components/ManaCalculator/components/AdvancedSettingsValidator.tsx

```diff
@@ -19,20 +19,33 @@ export function AdvancedSettingsValidator() {
   }
 
   return (
-    <Details summary='Advanced Settings - Validators'>
-      <div className='row'>
+    <Details
+      summary='Advanced Settings - Validators'
+      className='mana_calculator__card mana_calculator_inner__card table'
+    >
+      <div className='table'>
+        <div className='row'>
+          <div className='col col--2 text--center'>ID</div>
+          <div className='col col--2 horizontal-spaced text--center'>Stake</div>
+          <div className='col col--2 horizontal-space  text--center'>
+            Delegated
+          </div>
+          <div className='col col--2 horizontal-spaced text--center'>PF</div>
+          <div className='col col--2 horizontal-spaced text--center'>FC</div>
```

Comment:
Let's get rid of all my past mistakes with this PR 😆 

```suggestion
          <div className='col col--2 horizontal-spaced text--center'>Performance factor</div>
          <div className='col col--2 horizontal-spaced text--center'>Fixed costs</div>
```